### ISSUE TITLE
SVG badge embed in GitHub's readme

### ISSUE THREAD

**Issue by caktux** [rauchg/slackin#35]
![slackin-svg-issue](https://cloud.githubusercontent.com/assets/13692/7033946/ede72604-dd4b-11e4-9b88-46f74eec9323.png)

**Comment by xzyfer** [comment]
I was able to accomplish this with
```
[![Join us](https://subdomain.herokuapp.com/badge.svg)](https://subdomain.herokuapp.com/)
```

**Comment by caktux** [comment]
Try using it inline with other badges, the generated svg has the same height as its width, which causes the misalignment.

**Comment by caktux** [comment]
Looks like it was fixed in #28
